### DISCUSSION TITLE
Each line of redirected output is sent as a separate message

### DISCUSSION THREAD

**Discussion by 0xcrypto** [projectdiscovery/notify#31] [General]
In a multiline message, ie. 

```
$ cat /etc/passwd | notify --silent
```
Discord Webhook receives each line as a separate message. Is there a way to receive it as a single message?

**Comment by ehsandeep** [comment]
Hi @0xcrypto,

Each provider has limits on the size of the content we can post with one request, and we never know the size of the content in advance as the result request will fail or half content will be posted.

**Reply by 0xcrypto** [reply]
I just want to send a multiline message if notify can. Is there any way to do it? For now I am using discord.sh instead of notify.

**Comment by ehsandeep** [comment]
@0xcrypto this is now possible with Notify [v1.0.0 ](https://github.com/projectdiscovery/notify/releases/tag/v1.0.0)and file input with `bulk` flag.

**Reply by 0xcrypto** [reply]
Ah nice. Thanks
